src/indexset.jl

```diff
@@ -557,7 +557,7 @@ function replaceinds(is::Indices, rep_inds::Tuple{Vararg{Pair{<:Index,<:Index}}}
 end
 
 function replaceinds(is::Indices, rep_inds::Pair)
-  return replaceinds(is, Tuple(first(rep_inds)) .=> Tuple(last(rep_inds)))
+  return replaceinds(is, first(rep_inds) .=> last(rep_inds))
```

Comment:
I wonder if we can just use:
```julia
  return replaceinds(is, first(rep_inds), last(rep_inds))
```
instead, which would then directly call the core version `replaceinds(is::Indices, inds1, inds2)`. The logic of this code is a bit spaghetti-like, it's hard to track what is calling what and how it finally gets to the core implementation (which is my fault, this is some very early-days ITensors.jl code which has not been revisited in a while).